src/util.ts

```diff
@@ -280,3 +297,36 @@ export const addHashToBaseline = (hash: string, filepath: string): void => {
     ignoreMessages: baselineErrorsFile.meta.ignoreMessages
   })
 }
+
+export const toGitLabOutputFormat = (
+  errorSummaryMap: ErrorSummaryMap,
+  specificErrorMap: SpecificErrorsMap,
+  errorOptions: ErrorOptions
+): string => {
+  const result: GitLabErrorFormat[] = []
+
+  for (const [key, error] of errorSummaryMap.entries()) {
+    const specificErrors = getSpecificErrorsMatchingSummary(
+      error,
+      specificErrorMap,
+      errorOptions
+    )
+
+    const specificError: SpecificError = specificErrors[0] || {}
+
+    result.push({
+      description: error.message || 'Unknown error message',
+      check_name: 'typescript-errors',
+      fingerprint: key || 'unknown-fingerprint',
+      severity: 'minor',
+      location: {
+        path: specificError.file || 'unknown-file',
```

Comment:
The `file` is not optional. Why would we have a fallback for it?

Reply:
Yes, that makes sense and has been removed.